File: chapter02/akkademy-db-scala-2/src/main/scala/com/lp/akka/notes/RunApp.scala
package com.lp.akka.notes

import akka.actor.{ActorSystem, Props}

/**
 * @title TODO
 * @author li.pan
 * @version 1.0.0
 * @createTime 2021年04月16日 12:29:00
 *             <p>
 *             $description
 *             </p>
 */
object RunApp extends App {
  val system: ActorSystem = ActorSystem.create("akkademy")
  system.actorOf(Props[AkkademyDb], "akkademy-db")
}

File: chapter07/src/main/java/com/lp/akka/notes/cluster/wordcountv2/RunApp.java
package com.lp.akka.notes.cluster.wordcountv2;

import akka.actor.ActorRef;
import akka.actor.ActorSystem;
import akka.actor.Props;

/**
 * @author li.pan
 * @title
 */
public class RunApp {
    public static void main(String[] args) {
            ActorSystem system = ActorSystem.create("sys");

            ActorRef ref = system.actorOf(Props.create(ClusterPoolRouter.class), "statsService");

        }

}
